src/main/java/kr/codesquad/library/admin/common/PagingProperties.java

```diff
@@ -0,0 +1,71 @@
+package kr.codesquad.library.admin.common;
+
+import lombok.Getter;
+import org.springframework.data.domain.Page;
+
+import static kr.codesquad.library.admin.common.ConstantsCoveringMagicNumber.PAGE_GROUP_SIZE;
+
+@Getter
+public class PagingProperties {
+
+    private final int pageSize;
+    private final int currentPage;
+    private final int totalPages;
+    private final int pageGroupSize;
+    private final int totalPageGroups;
+    private final int currentPageGroup;
+    private final int startPageOfPageGroup;
+    private final int endPageOfPageGroup;
+    private final int endPageOfPreviousPageGroup;
+    private final int startPageOfNextPageGroup;
+
+    private PagingProperties(int pageSize, int currentPage, int totalPages) {
+        this.pageSize = pageSize;
+        this.currentPage = currentPage;
+        this.totalPages = totalPages;
+        this.pageGroupSize = PAGE_GROUP_SIZE;
+        this.totalPageGroups = calculateTotalPageGroups();
+        this.currentPageGroup = calculateCurrentPageGroup();
+        this.startPageOfPageGroup = calculateStartPageOfPageGroup();
+        this.endPageOfPageGroup = calculateEndPageOfPageGroup();
+        this.endPageOfPreviousPageGroup = calculateEndPageOfPreviousPageGroup();
+        this.startPageOfNextPageGroup = calculateStartPageOfNextPageGroup();
+    }
+
+    private int calculateTotalPageGroups() {
+        return (int) (Math.ceil((double) totalPages / pageGroupSize));
+    }
+
+    private int calculateCurrentPageGroup() {
+        return (int) (Math.ceil((double) currentPage / pageGroupSize));
+    }
+
+    private int calculateStartPageOfPageGroup() {
+        int startPageOfPageGroup = (currentPageGroup - 1) * pageGroupSize + 1;
+        if (startPageOfPageGroup > totalPages) { startPageOfPageGroup = totalPages; }
+        return startPageOfPageGroup;
+    }
+
```

Comment:
`Nit` 
```suggestion
    private int calculateStartPageOfPageGroup() {
        int startPageOfPageGroup = (currentPageGroup - 1) * pageGroupSize + 1;
        if (startPageOfPageGroup > totalPages) { 
            return totalPages; 
        }
        return startPageOfPageGroup;
    }
```

바로 리턴하면 되는데 굳이 값을 넣어서 if 문 밖으로 나갈 필요는 없는 듯 합니다.